pages/parent-enrollment.tsx

```diff
@@ -1,82 +1,74 @@
-                                    : "white"
-                            }
-                            lineHeight="24px"
-                            fontSize="16px"
-                            fontWeight="normal"
-                            textColor="#0C53A0"
-                            borderRadius="6px"
-                            height="50px"
-                            width="340px"
-                            key={childName}
-                            onClick={() => setSelectedChild(childName)}
-                            border={
-                                selectedChild === childName
-                                    ? null
-                                    : "2px solid #E1E1E1"
-                            }
-                        >
-                            {childName}
-                        </Button>
-                    ))}
-                </VStack>
-            </Center>
-            <Center mt="45px" mb="200px">
-                <Button
-                    height="50px"
-                    width="340px"
-                    borderRadius="6px"
-                    background={selectedChild === "" ? "#737373" : "#0C53A0"}
-                    fontWeight="normal"
-                    textColor="#FFFFFF"
-                    fontSize="16px"
-                    isDisabled={selectedChild === ""}
-                >
-                    Next
-                </Button>
-            </Center>
-        </Wrapper>
-    );
+export default function ParentEnrollClass(
+    props: ParentEnrollClassProps,
+): JSX.Element {
+    const [pageNum, setPagNum] = useState<number>(0);
+    const [selectedChild, setSelectedChild] = useState<number>(0);
+    const { user, isLoading, error } = useUser(props.session.id as string);
+
+    if (error) {
+        return <Box>{"An error has occurred: " + error.toString()}</Box>;
+    }
+    if (isLoading) {
+        return <Loading></Loading>;
+    }
+    if (user) {
+        const parentData = {
+            name: user.firstName + " " + user.lastName,
+            phone: user.parent.phoneNumber,
+        };
+        const studentData = user.parent.students;
+        const studentNames = user.parent.students.map((s) => {
+            return `${s.firstName} ${s.lastName}`;
+        });
+        console.log(studentData.dateOfBirth);
+        const pageElements = [
+            <SelectChildForClass
+                children={studentNames}
+                selectedChild={selectedChild}
+                setSelectedChild={setSelectedChild}
+                pageNum={pageNum}
+                setPageNum={setPagNum}
+            />,
+            <ClassEnrollmentConfirmation
+                studentData={studentData[selectedChild]}
+                parentData={parentData}
+                pageNum={pageNum}
+                setPageNum={setPagNum}
+            />,
+        ];
+        return <Wrapper>{pageElements[pageNum]}</Wrapper>;
+    }
+    return <Box></Box>;
```

Comment:
I think this is stale, at this point, if not error or loading, then user has to exist, this code is never used.
```suggestion
```